### PR TITLE
Update 13.2

### PR DESCRIPTION
**Pull request by IRacle1**
ill will code here for a new coming update, (after pr creating ill lost for 8 hours cuz sleep)

### CODE REVIEW

**Comment by IRacle1** [comment]
Ok, i fixed most of build/patch errors, except `Escaping` event, thats needs a full rewrite

**Comment by IRacle1** [comment]
ill do it tomorrow

**Comment by louis1706** [comment]
> Escaping

is fixed in an other PR

**Review by louis1706 — APPROVED** [review]

File: Exiled.API/Features/Server.cs [hunk 1]
```diff
@@ -175,8 +175,8 @@ public static float SpawnProtectTime
         /// </summary>
         public static bool IsHeavilyModded
         {
-            get => CustomNetworkManager.HeavilyModded;
-            set => CustomNetworkManager.HeavilyModded = value;
+            get => CustomNetworkManager.Modded;
+            set => CustomNetworkManager.Modded = value;
```
Comment (louis1706) [inline Exiled.API/Features/Server.cs:179]:
```suggestion
            get => ServerConsole.TransparentlyModdedServerConfig;
            set => ServerConsole.TransparentlyModdedServerConfig= value;
```

**Review by louis1706 — CHANGES_REQUESTED** [review]

**Review by louis1706 — APPROVED** [review]

**Pull request merged**